tools/releaseBuild/azureDevOps/templates/signBuildFiles.yml

```
@@ -0,0 +1,171 @@
+    }
+
+    ## Windows only modules
+
+    if('${{ parameters.addWindowsModules }}' -ne 'false') {
+      $itemsToCopy += @{
+        "$($fullSymbolsFolder)\pwsh.exe" = ""
+        "$($fullSymbolsFolder)\Microsoft.Management.Infrastructure.CimCmdlets.dll" = ""
+        "$($fullSymbolsFolder)\Microsoft.WSMan.*.dll" = ""
+        "$($fullSymbolsFolder)\Modules\CimCmdlets\CimCmdlets.psd1" = "Modules\CimCmdlets"
+        "$($fullSymbolsFolder)\Modules\Microsoft.PowerShell.Diagnostics\Diagnostics.format.ps1xml" = "Modules\Microsoft.PowerShell.Diagnostics"
+        "$($fullSymbolsFolder)\Modules\Microsoft.PowerShell.Diagnostics\Event.format.ps1xml" = "Modules\Microsoft.PowerShell.Diagnostics"
+        "$($fullSymbolsFolder)\Modules\Microsoft.PowerShell.Diagnostics\GetEvent.types.ps1xml" = "Modules\Microsoft.PowerShell.Diagnostics"
+        "$($fullSymbolsFolder)\Modules\Microsoft.PowerShell.Security\Security.types.ps1xml" = "Modules\Microsoft.PowerShell.Security"
+        "$($fullSymbolsFolder)\Modules\Microsoft.PowerShell.Diagnostics\Microsoft.PowerShell.Diagnostics.psd1" = "Modules\Microsoft.PowerShell.Diagnostics"
+        "$($fullSymbolsFolder)\Modules\Microsoft.WSMan.Management\Microsoft.WSMan.Management.psd1" = "Modules\Microsoft.WSMan.Management"
+        "$($fullSymbolsFolder)\Modules\Microsoft.WSMan.Management\WSMan.format.ps1xml" = "Modules\Microsoft.WSMan.Management"
+        "$($fullSymbolsFolder)\Modules\PSDiagnostics\PSDiagnostics.ps?1" = "Modules\PSDiagnostics"
+      }
+    }
+    else {
+      $itemsToCopy += @{
+        "$($fullSymbolsFolder)\pwsh" = ""
+      }
+    }
+
+    $itemsToExclude = @(
+      # This package is retrieved from https://www.github.com/powershell/MarkdownRender
+      "$($fullSymbolsFolder)\Microsoft.PowerShell.MarkdownRender.dll"
+    )
+
+    Write-Verbose -verbose "recusively copying $($itemsToCopyWithRecurse | out-string) to $filesToSignDirectory"
+    Copy-Item -Path $itemsToCopyWithRecurse -Destination $filesToSignDirectory -Recurse -verbose -exclude $itemsToExclude
+
+    foreach($pattern in $itemsToCopy.Keys) {
+      $destinationFolder = Join-Path $filesToSignDirectory -ChildPath $itemsToCopy.$pattern
+      $null = New-Item -ItemType Directory -Path $destinationFolder -Force
+      Write-Verbose -verbose "copying $pattern to $destinationFolder"
+      Copy-Item -Path $pattern -Destination $destinationFolder -Recurse -verbose
+    }
+  displayName: '${{ parameters.buildPrefixName }} - Prepare files to be signed'
+
+- template: EsrpSign.yml@ComplianceRepo
+  parameters:
+      buildOutputPath: $(System.ArtifactsDirectory)\toBeSigned
+      signOutputPath: $(System.ArtifactsDirectory)\signed
+      certificateId: "$(AUTHENTICODE_CERT)"
+      pattern: |
+        **\*.dll
+        **\*.psd1
+        **\*.psm1
+        **\*.ps1xml
+        **\*.ps1
+        **\*.exe
+      useMinimatch: true
+      shouldSign: $(SHOULD_SIGN)
+      displayName: ${{ parameters.buildPrefixName }} - Sign our binaries
```

This is a prefix, maybe make it shorter
```suggestion
      displayName: ${{ parameters.buildPrefixName }} - Authenticode
```